test/unit-tests/textBox.test.js

```diff
@@ -0,0 +1,32 @@
+let { openBrowser, goto, textBox, closeBrowser, intervalSecs, timeoutSecs } = require('../../lib/taiko');
+let { createHtml, removeFile, openBrowserArgs } = require('./test-util');
+
+describe('textBox', () => {
+    describe('with inline text', () => {
+        let filePath;
+        beforeAll(() => {
+            let innerHtml = '<form>' +
+                '<input type="text" name="color" value="red">Red</input>' +
+                '<input type="text" name="color" value="yellow">Yellow</input>' +
+                '<input type="text" name="color" value="greeen">Green</input>' +
+                '</form>';
+            filePath = createHtml(innerHtml);
+        });
+
+        beforeEach(async () => {
+            await openBrowser(openBrowserArgs);
+            await goto(filePath);
+        }, 10000);
+
+        afterAll(() => {
+            removeFile(filePath);
+        });
+
+        afterEach(async() => await closeBrowser());
+
+        test('test exists()', () => {
```

Comment:
tests can be added for the following cases: 

- input with type text and has label with for, inner element of label and using attribute/value pair  (exists, value)
(e.g) 
```
<label for="username">Username</label><input type="text" id="name" name="name"/>
<label>Do you like peas?
  <input type="checkbox" name="peas">
</label>
```
- textarea and has label with for, inner element of label and using attribute/value pair (exists, value)